webhook.go

```diff
@@ -0,0 +1,28 @@
+package monta
+
+// WebhookRequest is a webhook request from Monta.
+type WebhookRequest struct {
+	// List of webhook entries delivered in a batch.
+	Entries []WebhookEntry `json:"entries"`
+
+	// Number of pending events left for delivery.
```

Comment:
Does this mean the ones left not considering the entries delivered in this batch?

Reply:
That's what I understood from the docs.